src/backend/dataExports/openDataIndex.ts

```diff
@@ -30,7 +31,14 @@ export async function fetchOpenDataIndex(app: App) {
   for (const constantKey of openDataConstantKeys) {
     const key = app.constants[constantKey];
     const data = await s3GetJsonHelper(app.s3Client, { Bucket: app.constants.openDataBucket, Key: key });
-    openDataIndex[key] = data.meta;
+    // Fix the URL to point to the correct domain we are in
```

Comment:
:O I did not know fixing the meta link would require this extra step when putting the comment in your previous PR.

Thanks! : )